crates/cairo-lint-core/tests/test_files/bool_assert_comparison/bool_assert_comparison

```diff
@@ -0,0 +1,125 @@
+//! > assert_eq! with false
+
+//! > cairo_code
+// #[cfg(test)]
```

Comment:
do that for every test
```suggestion
#[cfg(test)]
```

Reply:
@0xLucqs  When I uncomment the #[cfg(test)] the diagnostics and the fixed fail. I just did push again so you can know what I'm doing wrong.

Reply:
ah yes you probably need to enable running the tests

Reply:
@0xLucqs I'm still having problems with (#[cfg(test)]) since as I told you it doesn't pass my tests like it should. I read your comment but I don't really understand how to enable the tests, that is, I execute the commands normally but beyond that I don't know if I should do anything extra with the tests.